File: app/src/main/java/app/endershrooms/thatcatapp/model/Image.java
package app.endershrooms.thatcatapp.model;

import androidx.room.ColumnInfo;
import com.squareup.moshi.Json;

public class Image {

  @Json(name = "id")
  @ColumnInfo(name = "img_id")
  private String id;
  @Json(name = "url")
  private String url;

  public String getId() {
    return id;
  }

  public void setId(String id) {
    this.id = id;
  }

  public String getUrl() {
    return url;
  }

  public void setUrl(String url) {
    this.url = url;
  }

}

File: app/src/main/java/app/endershrooms/thatcatapp/model/Image.kt
package app.endershrooms.thatcatapp.model

import androidx.room.ColumnInfo
import com.squareup.moshi.Json

data class Image(
    @Json(name = "id") @ColumnInfo(name = "img_id") var id: String?,
    @Json(name = "url") var url: String? = null
)